.travis.yml

```diff
@@ -1,11 +1,19 @@
-language: python
-install:
- - git clone https://github.com/w3c/respec.git
- - git clone https://github.com/dontcallmedom/webidl-checker.git
- - git clone https://github.com/dontcallmedom/widlproc
- - cd widlproc && make obj/widlproc && cd ..
- - pip install html5lib lxml
+language: node_js
+
+sudo: false
+
+branches:
+  only:
+    - TR
+
+env:
+  global:
+    - URL="https://rawgit.com/w3c/mediacapture-depth/TR/ECHIDNA"
```

Comment:
The rawgit.com URL works (well, at least last time we published) for the Presentation API with Echidna, so I believe this cannot be the reason for the failure?

https://github.com/w3c/presentation-api/blob/gh-pages/.travis.yml#L11